docker-compose.yml

```diff
@@ -41,16 +41,16 @@ services:
       - AGENT_JOBSTATUS=${AGENT_JOBSTATUS}
       - GIT_BRANCH=${GIT_BRANCH}
 
-  bg-geocode:
+  bg:
     build:
```

Comment:
Can we remove the build here? This uses the same image as `web`

Reply:
done....